### ISSUE TITLE
Fix contact page captcha

### ISSUE THREAD

**Issue by millserd** [CFLBajans/cfbajans-website-legacy#10]
Captcha is off and sending email doesn’t give acknowledgement for either correct or wrong answer.

This needs to be fixed or swapped with working captcha verification.

**Comment by kwhitehall** [comment]
Addressed in PR #17

**Comment by kwhitehall** [comment]
@millserd is this issue sorted?

**Comment by millserd** [comment]
Not quite. I prematurely closed this. I still need to push the changes I made that incorporate hiding/disabling of the submit button until the captcha has been verified/completed.
